packages/subproviders/test/integration/ledger_subprovider_test.ts

```diff
@@ -58,7 +58,9 @@ describe('LedgerSubprovider', () => {
             const data = ethUtils.bufferToHex(ethUtils.toBuffer('hello world'));
             const ecSignatureHex = await ledgerSubprovider.signPersonalMessageAsync(data);
             expect(ecSignatureHex.length).to.be.equal(132);
-            expect(ecSignatureHex.substr(0, 2)).to.be.equal('0x');
+            expect(ecSignatureHex).to.be.equal(
+                '0x1b0ec5e2908e993d0c8ab6b46da46be2688fdf03c7ea6686075de37392e50a7d7fcc531446699132fbda915bd989882e0064d417018773a315fb8d43ed063c9b00',
```

Comment:
Is this now guaranteed to be the same value for everyone whose set up their Ledger with the mnemonic?

Reply:
It should be deterministic as long as all of the inputs remain constant